Writing Day - Convince Community and Blog CTA (#2094)
writethedocs/www: docs/conf/portland/2024/news/call-for-projects-writing-day.rst

:template: {{year}}/generic.html
:og:image: /_static/conf/images/headers/{{shortcode}}-{{year}}-opengraph.jpg

.. post:: Feb 12, 2024
   :tags: {{shortcode}}-{{year}}, writing day, projects, oss projects

Announcing the Writing Day Call for Projects
============================================

Hey folks! We were so thrilled with your response to Writing Day last year that we are launching 
a `Call for Projects submission form <https://forms.gle/5WPN8MGCKzKqZBWU6>`_!

> For projects submitted by March 28, 2024, we have plans to highlight your information in our 
> pre-conference social media. We learned last year that projects that submit their information 
> pre-conference have higher attendance. In fact, perusing project details encourages documentarians 
> to join us earlier, stay longer, and check out multiple projects.

Reviewer: ```suggestion
For projects submitted by March 28, 2024, we'll highlight your project on our social media, as we learned last year that projects that submit their information before the conference have higher attendance. In fact, perusing project details encourages documentarians to join us earlier, stay longer, and check out multiple projects.
```

Our attendees are curious and want to know about you, your project, and your community - so 
let's give the people what they want!

You are of course welcome to submit and announce your project at Writing Day. This is a time honored 
tradition and we appreciate all of our projects.

Call for Projects
-----------------

Bringing your project to Writing Day is a wonderful way to introduce your mission and community to 
Write the Docs attendees.

The Write the Docs conference has many enthusiastic attendees. These attendees, documentarians, want 
to learn new technologies, sharpen their skills, and add to their portfolios. The world of open 
source has a seemingly endless number of wonderful open source projects. Documentarians want to write 
the docs, your open source project docs.

We are looking for community/OSS projects for our in-person Writing Day at the Portland conference on April 14.

The `Call for Proposals submission form <https://forms.gle/5WPN8MGCKzKqZBWU6>`_ is open until *11:59 PM, March 28, 2024 PST*.

> (If you’d prefer not to attend an in-person event due to COVID-19 or any other reason, keep an eye out for our 
> Atlantic conference held online in a timezone as widely accessible as possible).

Reviewer: ```suggestion
(If you’d prefer not to attend an in-person event due to COVID-19 or any other reason, keep an eye out for our 
Atlantic conference held online in timezone selected for accessibilty.
```
